openbas-api/src/main/java/io/openbas/rest/tag/TagApi.java

```diff
@@ -26,6 +31,10 @@
 import org.springframework.web.bind.annotation.*;
 
 @RestController
+@io.swagger.v3.oas.annotations.tags.Tag(
+    name = "Tags management",
+    description = "Endpoints to manage tags")
```

Comment:
Maybe we should describe here what tags are or put a link to the public doc as well ?

Reply:
Yes ! Unfortunately, there are no section on the docs about tags but I would be happy to add it once there is one.